dev/tasks/conda-recipes/appveyor.win.yml

```diff
@@ -1,20 +1,3 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Because these files are mostly ported from the [py]arrow[-cpp] feedstocks. There is a note in the LICENSE.txt that files under `dev/tasks/conda-recipes` have another license. Also added to the rat exclude list.